crates/matrix-sdk-crypto/src/store/types.rs

```diff
@@ -86,6 +86,8 @@ pub struct Changes {
     /// Historical room key history bundles that we have received and should
     /// store.
     pub received_room_key_bundles: Vec<StoredRoomKeyBundleData>,
+
+    pub sender_data: BTreeMap<OwnedRoomId, BTreeMap<String, KnownSenderData>>,
```

Comment:
this needs documentation at the very least, but I don't understand why it is needed and how it relates to `inbound_group_sessions`